srcpkgs/base-files/files/_vkpurge

```diff
@@ -0,0 +1,34 @@
+#compdef vkpurge
+
+local context state state_descr line
+typeset -A opt_args
+local curcontext="$curcontext"
+
+_arguments -C \
+	'1: :->subcmd' \
+	'*:: :->kernels'
+
+case "$state" in
+subcmd)
+	local subcommands=(
+		'list:List removable kernel versions'
+		'rm:Remove removable kernel versions'
```

Comment:
[The completion style guide](https://github.com/zsh-users/zsh/blob/master/Etc/completion-style-guide#L45) states that descriptions should not start with a capital letter. Not a requirement though.